packages/loot-core/src/types/models/payee.d.ts

```diff
@@ -0,0 +1,10 @@
+import type { AccountEntity } from './account';
+
+export interface PayeeEntity {
+  id?: string;
+  name: string;
+  transfer_acct?: AccountEntity;
+  tombstone?: boolean;
+  // TODO: remove once properly typed
+  [k: string]: unknown;
```

Comment:
@j-f1 @MatissJanis  I'd recommend either of you setting some time to properly think about these model types. I tried to get my head around them but I'm still confused 😅 . There are DB fields, models with snake_case, models with camelCase, hydrated models, client side models, migration models, ... I got lost.

Right now I've added this stopgap to all `*Entity` stuff so I could unblock myself, but I'd like you to use your knowledge of the system to come up (in another PR) with 2-3 proper, complete interfaces for each model that we can use everywhere.